### ISSUE TITLE
Incomplete version number in documentation site https://javaee.github.io/servlet-spec/

### ISSUE THREAD

**Issue by jorgefcduarte** [javaee/servlet-spec#181]
At https://javaee.github.io/servlet-spec/ page in servlet 3.1.0 maven dependency configuration version is incomplete.  If I put the same way as in the documentation page, my "mvn" can't download the resource. It worked when I added a '0' in the version number in pom.xml (3.1.0).

Not working pom dependency declaration (https://javaee.github.io/servlet-spec/):
```
<dependency>
  <groupId>javax.servlet</groupId>
  <artifactId>javax.servlet-api</artifactId>
  <version>3.1</version>
  <scope>provided</scope>
</dependency>
```

Working pom dependency declaration.

```
<dependency>
 <groupId>javax.servlet</groupId>
 <artifactId>javax.servlet-api</artifactId>
 <version>3.1.0</version>
  <scope>provided</scope>
</dependency>
```

**Comment by shingwaichan** [comment]
I have just fixed this one and another typo in the page.
Thanks.
     Shing Wai Chan

**Comment by shingwaichan** [comment]
commit a33c27f28623845c2c993d78691d6c2cb446b99e
Author: Shing Wai Chan <shing.wai.chan@oracle.com>
Date:   Tue Sep 26 10:02:22 2017 -0700

    fix typos in Servlet 3.1 section
